source/frontend/src/features/leases/detail/LeasePages/details/LeaseDetailView.tsx

```diff
@@ -22,7 +23,33 @@ export const LeaseDetailView: React.FunctionComponent<
   return (
     <Section header="Details">
       <SectionField label="Ministry project">{projectName}</SectionField>
-      <SectionField label="Status" labelWidth="3" contentWidth="4">
+      <SectionField
+        label="Status"
+        labelWidth="3"
+        contentWidth="4"
+        tooltip={
+          <TooltipIcon
+            toolTipId="lease-status-tooltip"
+            toolTip={
+              <ul>
+                <li>Draft: In progress but not finalized.</li>
```

Comment:
Not a change necessary, but if this tooltip is reused again we should either create a common tooltip for lease status or create an html component that contains the repeated tooltip information.